stores/exploreFilters.ts

```diff
@@ -0,0 +1,45 @@
+import { defineStore } from 'pinia'
+
+interface State {
+  // status
+  listed: boolean
+  owned: boolean
+  // price
+  min: number | undefined
+  max: number | undefined
+}
+
+export const useExploreFiltersStore = defineStore('exploreFilters', {
+  state: (): State => ({
+    listed: false,
+    owned: false,
+    min: undefined,
+    max: undefined,
+  }),
+  getters: {
+    getListed: (state) => state.listed,
```

Comment:
> I don't think we need `getListed, getOwned, getMin, getMax`. because we can access it directly. wdyt?

Yes! Should work like this too lemme change that